website/content/community-tools.mdx

```diff
@@ -19,7 +19,7 @@ vendors. These plugins are not officially tested nor officially maintained by
 HashiCorp, and are listed here in order to help users find them easily.
 
 To learn more about how to use community plugins, or how to build your own,
-check out the docs on [extending Packer](/packer/docs/plugins/install-plugins)
+check out the docs on [extending Packer](https://developer.hashicorp.com/packer/docs/plugins/install-plugins)
```

Comment:
This page isn't served in Developer yet, so this needs to remain an absolute link.

Reply:
hmm... I'm a little confused by the comment. What do you mean that it is not served in Developer yet?

Reply:
This page is[ still on .io,](https://www.packer.io/community-tools) so the relative link won't work!

Reply:
There's talk of integrating community pages / hubs like this on Developer in the future, so that's where the 'yet' came from 😄

Reply:
Actually although the link is still on `packer.io`, it does redirect automatically to `develop.hashicorp` so the link is not broken per se, but it is indeed not accurate